src/sections/Docker-Meshery/dockerMeshery.style.js

```diff
@@ -92,4 +92,8 @@ export const DockerMesheryWrapper = styled.div`
       padding: 5%;
       border-radius: 1rem;
     }
+    .jFTZLO {
```

Comment:
This won't work, as this is a temp/random className generated at the time of bundling.
Please use the actual className that the component has, or if needed add a new className to it.